app/javascript/components/landing-page.vue

```diff
@@ -0,0 +1,12 @@
+<template>
+<div>
+    <br>
+    <h3>¡¡¡ Welcome to KitchenGram !!!</h3>
+</div>
+</template>
+
+<script>
+
+export default {
+};
+</script>
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>  Newline required at end of file but not found        eol-last